bin/console

```diff
@@ -12,8 +12,8 @@ use Symfony\Component\Debug\Debug;
 
 set_time_limit(0);
 
-require __DIR__.'/../vendor/autoload.php';
-require_once __DIR__.'/../autoload.php';
+require_once __DIR__.'/../vendor/autoload.php';
```

Comment:
```suggestion
```
Already loaded in config.inc.php

Reply:
I know but without it, I have the feeling it won't works.

Let me remove it and let's see if this file is required or not :)